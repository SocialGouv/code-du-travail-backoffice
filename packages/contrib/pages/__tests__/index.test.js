import React from "react";
import { render } from "react-testing-library";
import Index from "../index";

describe("[Contrib] pages/<Index />", () => {
  it("should render", () => {
    const { container } = render(<Index />);

    expect(container).toMatchSnapshot();
  });
});
