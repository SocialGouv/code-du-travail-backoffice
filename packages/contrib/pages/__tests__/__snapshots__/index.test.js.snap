// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Contrib] pages/<Index /> should render 1`] = `
<div>
  <div
    class="sc-bdVaJa sc-bwzfXH Main__Container-q4q6nx-0 bBIrgk"
    style="height: 100vh;"
  >
    <div
      class="sc-bdVaJa sc-bwzfXH Main__Body-q4q6nx-1 eBCYWC"
      width="1"
    >
      <div
        class="sc-bdVaJa sc-bwzfXH Header__Container-sc-7cr55z-0 kivzys"
      >
        <div
          class="sc-bdVaJa sc-bwzfXH Header__Brand-sc-7cr55z-1 ccbJhB"
        >
          <img
            alt="Code du travail numérique"
            class="Header__Logo-sc-7cr55z-2 kAIypM"
            src="[object Object]"
          />
          <div
            class="sc-bdVaJa sc-bwzfXH eVPJMi"
          >
            <span
              class="Header__Title-sc-7cr55z-3 jGdlvq"
            >
              Code du travail numérique
            </span>
            <span
              class="Header__Subtitle-sc-7cr55z-4 kxPxIS"
            >
              Outil de contribution
            </span>
          </div>
        </div>
      </div>
      <div
        class="sc-bdVaJa sc-bwzfXH Main__Content-q4q6nx-2 lhyucv"
      >
        <div
          class="sc-bdVaJa sc-bwzfXH pages__Content-sht9xz-0 xAtXf"
          width="1"
        >
          <p
            class="Title-r6qlw8-0 kDtypV"
          >
            Mes réponses en cours de rédaction
          </p>
          <p>
            Aucune réponse trouvée.
          </p>
          <p
            class="Title-r6qlw8-0 kDtypV"
          >
            Réponses à rédiger
          </p>
          <p>
            Aucune réponse trouvée.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
